lib/puppet_litmus/rake_helper.rb

```diff
@@ -224,15 +224,26 @@ def install_module(inventory_hash, target_node_name, module_tar, module_reposito
     include ::BoltSpec::Run
 
     target_nodes = find_targets(inventory_hash, target_node_name)
-    bolt_result = upload_file(module_tar, File.basename(module_tar), target_nodes, options: {}, config: nil, inventory: inventory_hash.clone)
+    bolt_result = upload_file(module_tar, "/tmp/#{File.basename(module_tar)}", target_nodes, options: {}, config: nil, inventory: inventory_hash.clone)
```

Comment:
Curious on the choice to change the upload destination, is this required?

Reply:
I will test without this, i think this is working without this change.